files/jquery.quicksearch/update.json

```diff
@@ -0,0 +1,9 @@
+
+{
+  "packageManager": "github",
+  "name": "jquery.quciksearch",
+  "repo": "DeuxHuitHuit/quicksearch",
+  "files": {
+    "include": ["jquery.quicksearch.min.js", "jquery.quicksearch.min.js.map"]
```

Comment:
You need to add the basepath parameter set to `dist/` because the files are not in root
https://github.com/DeuxHuitHuit/quicksearch/tree/2.0.5/dist